### DISCUSSION TITLE
Empty vehicle.

### DISCUSSION THREAD

**Discussion by noahjsc** [voxeltycoon/issues#541] [UI&UX]
We should be able to force a vehicle to destroy its contents with a button. This would be really useful for when it gets loaded with the wrong cargo and has nowhere to unload.

**Comment by katepolak** [comment]
This can already be done using a workaround of sending it to a depot, opening the configuration screen and changing the cargo type.
But a proper way would definitely be better.

**Comment by victorleblais** [comment]
You can also add a refit order to your route before your load orders to make sure it loads the correct material. Example:
![image](https://user-images.githubusercontent.com/4365075/115540797-b16a5380-a29e-11eb-97fa-1f5c13b31721.png)
